src/main/java/com/turbinekreuzberg/plugins/utils/PhpContentCreator.java

```diff
@@ -26,6 +26,12 @@ public String create(PsiFile file, String relativePath) {
         contentWithClassName = contentWithClassName.replace("{{type}}", getType(file));
         contentWithClassName = contentWithClassName.replace("{{sprykerNamespace}}", sprykerNamespace);
 
+        if (!AppSettingsState.getInstance().fileDocBlockText.isBlank()) {
```

Comment:
This will break if `fileDocBlockText` is not set at all. I think we need to either adjust this statement or set a default value in `AppSettingsState`.
Also, if it's not set we need to remove the placeholder and the empty lines (or rather add the lines inside here).